src/components/header/modals/ModalNetwork.vue

```diff
@@ -21,30 +18,13 @@
                   name="choose_networks"
                   type="radio"
                   :checked="selNetwork === index"
-                  class="
-                    ip--network
-                    tw-appearance-none
-                    tw-border-2
-                    tw-border-gray-100
-                    tw-rounded-full
-                    tw-h-4
-                    tw-w-4
-                    tw-mr-2
-                    focus:tw-outline-none
-                    tw-bg-white
-                    checked:tw-border-4
-                  "
+                  class="ip--network"
```

Comment:
Please fix this
![image](https://user-images.githubusercontent.com/92044428/161742739-aa60328a-3987-44e3-a50b-f86ae3371250.png)